pyproject.toml

```diff
@@ -1,3 +1,115 @@
+[project]
+name = "cf_xarray"
+description = "A convenience wrapper for using CF attributes on xarray objects"
+readme = "README.rst"
+requires-python = ">=3.8"
+license = {file = "LICENSE"}
+keywords = ["xarray", "metadata", "CF conventions"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: Apache Software License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+]
+dependencies = [
+    "xarray",
+]
+dynamic = ["version"]
+
+[project.urls]
+homepage = "https://cf-xarray.readthedocs.io"
+documentation = "https://cf-xarray.readthedocs.io"
+repository = "https://github.com/xarray-contrib/cf-xarray.git"
+changelog = "https://github.com/xarray-contrib/cf-xarray/blob/main/doc/whats-new.rst"
+
+[build-system]
+requires = [
+    "xarray",
+    "setuptools>=45",
+    "wheel",
+    "setuptools_scm[toml]>=6.2",
+    "setuptools_scm_git_archive",
```

Comment:
If you are writing the version to a file I'm almost certain that you don't need this.